Formula/rex.rb

```diff
@@ -11,7 +12,14 @@ class Rex < Formula
     sha256 "d02002fba499b2a3ed5271ccf1c3ddbaf75629d747aed289a4f53de7aad9d081" => :yosemite
   end
 
-  depends_on :perl => "5.16"
```

Comment:
If it's not flagged by `brew audit`/`brew style` it's not worth changing in a random revision bump.